test/templates/spec/spec_helper.js

```diff
@@ -0,0 +1,23 @@
+(function() {
+  var SpecHelper = {
+    triggerEvent: function(el, eventName) {
+      var event = document.createEvent("MouseEvents");
+      event.initEvent(eventName, true, true); // type, bubbles, cancelable
+      el.dispatchEvent(event);
+    },
+    randomString: function() {
+      Math.random().toString(36).substring(2);
+    }
+  };
+
+  var stageElement = document.createElement("div");
```

Comment:
Is that something that is needed for every test we have? We may make it callable and include it only where needed.